offchain/rollups-events/src/broker/mod.rs

```diff
@@ -285,6 +354,7 @@ pub struct BrokerCLIConfig {
 #[derive(Debug, Clone)]
 pub struct BrokerConfig {
     pub redis_endpoint: RedactedUrl,
+    pub redis_cluster_endpoints: Vec<RedactedUrl>,
```

Comment:
It is strange to have both options when only one is available. The type does not correspond to the semantics of the object. We should use an enum to represent these fields. For instance:
```
enum BrokerEndpoint {
    Single(RedactedUrl),
    Cluster(Vec<RedactedUrl>),
}
```
Then
```suggestion
    pub endpoint: BrokerEndpoint,
```